llama-index-server/llama_index/server/server.py

```diff
@@ -10,23 +10,51 @@
 from llama_index.server.api.routers.chat import chat_router
 from llama_index.server.chat_ui import download_chat_ui
 from llama_index.server.settings import server_settings
+from pydantic import BaseModel, Field
+
+
+class UIConfig(BaseModel):
+    enabled: bool = Field(default=True, description="Whether to enable the chat UI")
+    app_title: str = Field(
+        default="LlamaIndex Server", description="The title of the chat UI"
+    )
+    starter_questions: Optional[list[str]] = Field(
+        default=None, description="The starter questions for the chat UI"
+    )
+    ui_path: str = Field(
+        default=".ui", description="The path that stores static files for the chat UI"
+    )
+    use_llamacloud: bool = Field(
```

Comment:
use_llamacloud is not a UI parameter.

we need `llamacloud_index_selector` with default `False`

in the generated create-llama code we want it to set to `llamacloud_index_selector=False` for both llama cloud and non llamacloud - means in both cases the selector is not shown (and means the index from the env variable is used for llamacloud)